pkgs/servers/sql/materialize/default.nix

```
@@ -40,17 +40,17 @@ let
 in
 rustPlatform.buildRustPackage rec {
   pname = "materialize";
-  version = "0.9.4";
-  rev = "29d003cae5e9d46f8b11b2102ff0b9abf6608c2f";
+  version = "0.10.0";
+  rev = "c14633f59e842fbdd62c7239ffd8c2a16704386e";
```

```suggestion
  MZ_DEV_BUILD_SHA = "c14633f59e842fbdd62c7239ffd8c2a16704386e";
```
We could do that to make it obvious where rev is used.

---

good idea, thanks! It was like that from an earlier version that used `rev` in fetchFromGithub. Fixed